### PR TITLE
Add fd, ripgrep, hyperfine

### PR DESCRIPTION
**Pull request by dbast**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

3 cli tools in one PR, all Rust, all follow the same recipe pattern. Making them available cross platform.

### CODE REVIEW

File: recipes/fd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{% set version = "7.3.0" %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: fd
```
Comment (bgruening) [inline recipes/fd/meta.yaml]: There is a unix utility with the same name. In debian in the fdclone package. Maybe we should change the name of the package?

Comment (dbast) [reply recipes/fd/meta.yaml]: Well, that is the problem of two letters only... Renaming the package or command would make it cumbersome to find it and would also then deviate from the documentation or how it is already named in e.g. homebrew, https://github.com/Homebrew/homebrew-core/blob/master/Formula/fd.rb

If you have a good idea here, feel free to update the PR.

**Comment by dbast** [comment]
@conda-forge/staged-recipes Ready for review. Feel free to update the PR. More co-maintainers welcome. Will be afk for some days. Thanks

File: recipes/fd/meta.yaml (hunk 1 above)
Comment (chrisburr) [reply recipes/fd/meta.yaml]: Looking around at other package managers it seems like the consensus is either `fd` (Homebrew, Arch, openSUSE, Slackware) or `fd-find` (Ubuntu, Fedora, Debian, Free BSD).

Comment (dbast) [reply recipes/fd/meta.yaml]: @chrisburr Thanks for looking that all up! Naming the packing fd-find sounds like a good idea. Will rename it after being back from afk (or I manage to do so with working copy on a tiny iPhone). Any other findings? Thanks

Comment (chrisburr) [reply recipes/fd/meta.yaml]: Nope it looks good, I've edited the name so LGTM 👍

**Comment by dbast** [comment]
@chrisburr thanks!

**Pull request merged**